### PR TITLE
Update configmap to set roks parameters

### PR DESCRIPTION
**Pull request by yannizhang2019**
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/39426

### CODE REVIEW

File: pkg/controller/authentication/configmap.go [hunk 1]
```diff
@@ -45,40 +52,37 @@ func (r *ReconcileAuthentication) handleConfigMap(instance *operatorv1alpha1.Aut
 	err = r.client.Get(context.TODO(), types.NamespacedName{Name: consoleConfigMapName, Namespace: instance.Namespace}, consoleConfigMap)
 	if err != nil {
 		if errors.IsNotFound(err) {
-			reqLogger.Error(err, "The configmap ", consoleConfigMapName ," is not created yet")
-			return err
-		} else {
-			reqLogger.Error(err, "Failed to get ConfigMap",  consoleConfigMapName)
+			reqLogger.Error(err, "The configmap ", consoleConfigMapName, " is not created yet")
```
Comment (posriniv) [inline pkg/controller/authentication/configmap.go:55]: @yannizhang2019 why did you remove the else block here

File: pkg/controller/authentication/configmap.go [hunk 2]
```diff
@@ -45,40 +52,37 @@ func (r *ReconcileAuthentication) handleConfigMap(instance *operatorv1alpha1.Aut
...
 	proxyConfigMapName := "ibmcloud-cluster-info"
 	proxyConfigMap := &corev1.ConfigMap{}
 	err = r.client.Get(context.TODO(), types.NamespacedName{Name: proxyConfigMapName, Namespace: instance.Namespace}, proxyConfigMap)
 	if err != nil {
 		if errors.IsNotFound(err) {
-			reqLogger.Error(err, "The configmap ", proxyConfigMapName ," is not created yet")
-			return err
-		} else {
-			reqLogger.Error(err, "Failed to get ConfigMap",  proxyConfigMapName)
+			reqLogger.Error(err, "The configmap ", proxyConfigMapName, " is not created yet")
```
Comment (posriniv) [inline pkg/controller/authentication/configmap.go:69]: @yannizhang2019 why did you remove the else block here?

File: pkg/controller/authentication/configmap.go (hunk 1 above)
Comment (yannizhang2019) [reply pkg/controller/authentication/configmap.go:55]: that's from go lint

File: pkg/controller/authentication/configmap.go (hunk 2 above)
Comment (yannizhang2019) [reply pkg/controller/authentication/configmap.go:69]: go lint flagged it, so I changed based on go lint recommendation, we still have a lot go lint errors
